Formula/b2sum.rb

```diff
@@ -0,0 +1,20 @@
+class B2sum < Formula
+  desc "BLAKE2 b2sum reference binary"
+  homepage "https://github.com/BLAKE2/BLAKE2"
+  url "https://github.com/BLAKE2/BLAKE2/archive/20160619.tar.gz"
+  sha256 "cbac833ccae56b5c6173dbeaf871aa99b32745cf7a994c7451d4533ecda55633"
+
+  def install
+    cd "b2sum" do
+      # https://github.com/BLAKE2/BLAKE2/issues/25
+      system "make", "NO_OPENMP=1"
+      bin.install "b2sum"
+      man1.install "b2sum.1"
```

Comment:
Using the `install` target helps us stay in sync with upstream (e.g. if a new file is added), though I admit that this is unlikely to happen in this case.

But no need to further discuss this, as there is a somewhat usable `install` target. Sadly, the dependencies between targets are not properly declared, so we can't just run `make install` to get both compilation and installation in one step. Here's what should work (not tested myself):

```ruby
system "make", "NO_OPENMP=1"
system "make", "install", "PREFIX=#{prefix}", "MANDIR=#{man}"
```